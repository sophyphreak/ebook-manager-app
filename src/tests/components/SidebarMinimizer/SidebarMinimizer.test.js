import React from 'react';
import { shallow } from 'enzyme';
import SidebarMinimizer from '../../../components/SidebarMinimizer/SidebarMinimizer';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<SidebarMinimizer />);
});

test('should render SidebarMinimizer correctly', () => {
  expect(wrapper).toMatchSnapshot();
});