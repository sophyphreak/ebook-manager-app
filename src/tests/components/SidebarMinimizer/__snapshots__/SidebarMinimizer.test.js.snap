// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SidebarMinimizer correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SidebarMinimizer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "sidebar-minimizer",
      "onClick": [Function],
      "type": "button",
    },
    "ref": null,
    "rendered": null,
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "sidebar-minimizer",
        "onClick": [Function],
        "type": "button",
      },
      "ref": null,
      "rendered": null,
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
